code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -148,14 +151,22 @@
 
 /datum/reagent/medicine/kelotane
 	name = "Kelotane"
-	description = "Kelotane is a drug used to treat burns."
+	description = "Kelotane is a drug used to treat burns. More effective in higher dosage, but causes slight pain."
 	color = "#D8C58C"
 	scannable = TRUE
 	overdose_threshold = REAGENTS_OVERDOSE
 	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL
 
 /datum/reagent/medicine/kelotane/on_mob_life(mob/living/L, metabolism)
+	var/target_temp = L.get_standard_bodytemperature()
+	if(L.bodytemperature > target_temp)
+		L.adjust_bodytemperature(-5 * TEMPERATURE_DAMAGE_COEFFICIENT, target_temp) //smol cooldown to make fire less-bad.
```

Comment:
If you set the min and max values you don't need to check for the body temperature.

Reply:
One, god willing, we might get non-human marines one day.
Two, seemed more compact.

Lemme know if you want me to change it.